### DISCUSSION TITLE
Received security email, can't sign in due to lost 2FA device, can't contact support because contacting support requires signing in

### DISCUSSION THREAD

**Discussion by darkpixel** [npm/feedback#695] [Website] [answered]
I received the recent security email about tokens being potentially compromised.
I figured I should sign in to my account, rotate stuff, update my password, and enable 2FA...but the site said my password was invalid.

I used the "forgot password" link, and started to go through the process, and it asked me for my 2FA code.

Oops.  I already had 2FA enabled from years ago, but the phone that had the code was dropped in a puddle a while back.

Went into the docs and found the article on 2FA recovery.

It said to "contact support".
Went to "contact support" and it says I have to be signed in to contact support.
Can't sign in.

**Comment by MylesBorins** [comment, marked as answer]
@darkpixel you can still submit a form via the support link even if you are not signed in

1. visit https://www.npmjs.com/support
2. Under "Can't sign in, or don't have an account" click "Account or billing issue" (you may need to scroll down)
3. Fill out the form
